malaria24/ona/tests/test_models.py

```diff
@@ -496,6 +496,145 @@ def test_send_digest_email(self):
         self.assertEqual(html_content.count('EHP1, EHP2'), 10)
         self.assertEqual(message.to, [mis.email_address])
 
+    @responses.activate
+    def test_print_district_digest_email_data(self):
```

Comment:
Since you're not printing anything I'd recommend changing this test name to something like "test_district_digest_email_data"